code/modules/tables/presets.dm

```diff
@@ -39,6 +39,15 @@
 	table_mat = MATERIAL_WOOD
 	table_reinf = MATERIAL_WOOD
 
+/obj/structure/table/reinforced/marbleglass
+	table_mat = MATERIAL_GLASS
+	table_reinf = MATERIAL_MARBLE
+	alpha = 77
```

Comment:
why?

Reply:
I was trying to get a reinforced design to show up on a glass table, but it didn't work out, so I'll just use another kind of table instead.  Plus, reinforced glass tables!
(The bronze-reinforced wooden table looks badass though.)

Reply:
Removed.